proxy.go

```diff
@@ -66,23 +69,27 @@ func (ph ProxyHandler) handleConnect(w http.ResponseWriter, req *http.Request) {
 		w.WriteHeader(http.StatusBadGateway)
 		return
 	}
+	// Establish a connection back to the client, by hijacking the ResponseWriter.
 	h, ok := w.(http.Hijacker)
 	if !ok {
-		msg := fmt.Sprintf("Can't hijack connection to %v", req.Host)
-		http.Error(w, msg, http.StatusInternalServerError)
+		log.Printf("[%d] Can't hijack response writer", id)
```

Comment:
Likewise, this could be "Error converting response writer into a hijacker"

Reply:
Done